cmd/coco/generate/renderTemplate.go

```diff
@@ -0,0 +1,103 @@
+package generate
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	gotemplate "text/template"
+
+	"github.com/SAP/configuration-tools-for-gitops/pkg/files"
+	"github.com/SAP/configuration-tools-for-gitops/pkg/yamlfile"
+)
+
+var (
+	filesWrite = files.Write
+)
+
+func ParseTemplate(filename string, valueFiles []string, target string) error {
+	p := parser{}
+	if err := p.parse(filename, tmplFuncs()); err != nil {
```

Comment:
why not call tmplFuncs() directly inside parse() since we are always passing the same function?

Reply:
addressed